src/tests/Microsoft.VisualStudio.Composition.Tests/XunitExtensions/AssertFailedException.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace Microsoft.VisualStudio.Composition.Tests
+{
+    using System;
+    using System.Runtime.Serialization;
+
+    [Serializable]
+    internal class AssertFailedException : Exception
+    {
+        public AssertFailedException()
```

Comment:
nit: can you use the `internal` modifier on members that are actually internal (by virtue of their internal container)? That way, my public API cost alarm doesn't go off during code reviews. :)